### ISSUE TITLE
QuickPick not accessible with VoiceOver

### ISSUE THREAD

**Issue by isidorn** [microsoft/vscode#104636]
1. Have latest VS Code insiders, turn on Voice Over on mac
2. cmd + P -> notice how the screen reader does not read any of the items in the quick pick

This works just fine on Linux and Windows.
Also works fine for stable. So it seems like an chrome update in electron broke this
I can also repro running `yarn web` in `Chrome`. In Firefox and Safari all is good.
I can not reproduce in Chrome cannary, so there seems to be a fix in Chrome latest.

Ideally we would find the fix in Chrome and backport it to our Electron.

fyi @deepak1556 @bpasero 
And @joanmarie for Chrome smartness, maybe one of your collegues fixed something like this (should be related to `activeDescendent`)

**Comment by joanmarie** [comment]
I can add this to my list (which is starting to grow I'm afraid). BUT, you don't need smartness; you need the most excellent bisect-build tool! :smile: Docs here: https://sites.google.com/a/chromium.org/dev/developers/bisect-builds-py. Once you have the regression range, you just sit there as pre-built binaries are presented for you to try and mark as good or bad. Best. Tool. Ever.

Lemme know if you have any questions or need help.

**Comment by isidorn** [comment]
Thanks a lot. Let's see if @deepak1556 already has this set up, otherwise I can also look into it.

**Comment by deepak1556** [comment]
Yup I can do the bisect shortly.

**Comment by deepak1556** [comment]
Bisect result: 
```
You are probably looking for a change made after 722020 (known good), but no later than 722031 (first known bad).
CHANGELOG URL:
  https://chromium.googlesource.com/chromium/src/+log/840ebe6dc078c78651d7beebf06b29d81039f882..a7d3657ad4c762ba4829a9c6f7677cd92f096394
```

Potential candidate: https://chromium-review.googlesource.com/c/chromium/src/+/1949533

**Comment by deepak1556** [comment]
Looking for fix now.

**Comment by deepak1556** [comment]
Upstream fix: https://chromium-review.googlesource.com/c/chromium/src/+/2303789
Crbug: https://bugs.chromium.org/p/chromium/issues/detail?id=1082865

**Comment by isidorn** [comment]
Impressive. Thanks a lot!
